core/src/main/scala/kafka/network/SocketServer.scala

```diff
@@ -594,6 +614,9 @@ private[kafka] object Processor {
   val IdlePercentMetricName = "IdlePercent"
   val NetworkProcessorMetricTag = "networkProcessor"
   val ListenerMetricTag = "listener"
+  val SocketServerMetricsGroup = "socket-server-metrics"
```

Comment:
Should this be in the `SocketServer` companion object? It could then just be called `MetricsGroup` perhaps.

Reply:
@ijuma Thanks for the review, updated.